Octokit/Models/Request/IssueRequest.cs

```diff
@@ -121,17 +121,17 @@ public enum IssueFilter
     }
 
     /// <summary>
-    /// The range of states that an issue can be in.
+    /// Search API Filter
     /// </summary>
-    public enum ItemState
+    public enum ItemStateFilter
     {
         /// <summary>
-        /// Isuses that are open (default).
+        /// Issues that are open (default).
```

Comment:
Perhaps say "Items" rather than "Issues"

Also Im not sure if we should say "(default)" here, since the default behaviour would technically be based on the place of usage (eg on the ```State``` field of ```IssueRequest``` ```PullRequestRequest``` etc is where you could mention what the default value is for that operation)